pontoon/api/schema.py

```diff
@@ -0,0 +1,117 @@
+import graphene
+from graphene_django import DjangoObjectType
+from graphene_django.debug import DjangoDebug
+
+from pontoon.api.util import get_fields
+
+from pontoon.base.models import (
+    Project as ProjectModel,
+    Locale as LocaleModel,
+    ProjectLocale as ProjectLocaleModel
+)
+
+
+class ProjectLocale(DjangoObjectType):
+    class Meta:
+        model = ProjectLocaleModel
+        only_fields = (
+            'total_strings', 'approved_strings', 'translated_strings',
+            'fuzzy_strings', 'missing_strings', 'complete', 'project', 'locale'
+        )
+
+
+class Project(DjangoObjectType):
+    class Meta:
+        model = ProjectModel
+        only_fields = (
+            'name', 'slug', 'disabled', 'info', 'deadline', 'priority',
+            'contact', 'total_strings', 'approved_strings',
+            'translated_strings', 'fuzzy_strings', 'missing_strings',
+            'complete'
+        )
+
+    localizations = graphene.List(ProjectLocale)
+
+    @graphene.resolve_only_args
+    def resolve_localizations(obj):
+        return obj.project_locale.all()
+
+
+class Locale(DjangoObjectType):
+    class Meta:
+        model = LocaleModel
+        only_fields = (
+            'name', 'code', 'direction', 'script', 'population',
+            'total_strings', 'approved_strings', 'translated_strings',
+            'fuzzy_strings', 'missing_strings', 'complete'
+        )
+
+    localizations = graphene.List(ProjectLocale)
+
+    @graphene.resolve_only_args
+    def resolve_localizations(obj):
+        return obj.project_locale.all()
+
+
+class Query(graphene.ObjectType):
+    debug = graphene.Field(DjangoDebug, name='__debug')
+
+    projects = graphene.List(Project, with_disabled=graphene.Boolean(False))
```

Comment:
Can you add a comment that explains that this with_disabled thing is a default value for queries? I assume that's what it does because of what you told me earlier, but it isn't obvious looking at the code.

Reply:
And here's the proof that a comment is needed: I didn't understand what that was actually doing. I think I would call that argument ``include_disabled`` instead, to make very clear what it does. And please do add a comment too. :)